src/main/java/mekanism/common/item/gear/ItemGasArmor.java

```diff
@@ -13,6 +13,7 @@
 import mekanism.common.item.interfaces.IGasItem;
 import mekanism.common.util.ChemicalUtil;
 import mekanism.common.util.StorageUtils;
+import net.minecraft.client.gui.screens.Screen;
```

Comment:
??

Reply:
Ups, I thought I changed the use of `Scree#hasShiftDown` to `TooltipFlag#isAdvanced`. Fixed.